examples/language_identification_example.py

```diff
@@ -18,18 +18,22 @@
 
 
 # String containing your access token
+from rev_ai.models.customer_url_data import CustomerUrlData
+
 access_token = "<your_access_token>"
 
 # Create your api client
 client = language_identification_client.LanguageIdentificationClient(access_token)
 
 # Submitting a job with a link to the file you want to identify the language of
 # Change url to your url
```

Comment:
I don't think this comment makes sense with the new source_config. We are encouraging people to use source config and notification config instead of  media and callback urls

Reply:
Yeah it could be more clear. Updated with `# Change the url specified in source_config to the url of the media to submit`